internal/lsp/server.go

```diff
@@ -434,7 +488,7 @@ func (l *LanguageServer) handleTextDocumentCodeAction(
 
 	for _, diag := range params.Context.Diagnostics {
 		switch diag.Code {
-		case "opa-fmt":
+		case ruleNameOPAFmt:
```

Comment:
We should do this for the v1 rule too, but I guess it's not rebase here?

Reply:
I am lazy, golangci lint didn't flag that one... added in d5bee69